universite-du-quebec-a-montreal.csl

```diff
@@ -725,6 +1118,26 @@
       </else>
     </choose>
   </macro>
+  <macro name="number">
+    <choose>
+      <if type="legal_case" match="all" variable="references">
+        <text variable="number"/>
+      </if>
+      <else-if type="legal_case" match="all" variable="publisher">
+        <text variable="number"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <group prefix=" (" suffix=")">
+      <date form="text" variable="original-date">
+        <date-part name="year"/>
+        <date-part name="year"/>
+        <date-part name="year"/>
+      </date>
+      <text variable="year-suffix"/>
+    </group>
+  </macro>
   <citation et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" givenname-disambiguation-rule="primary-name" collapse="year">
```

Comment:
don't duplicate the subsequent et al here.

Reply:
OK